### ISSUE TITLE
`ep show`: "'generator' object has no attribute 'next'"

### ISSUE THREAD

**Issue by wcooley** [dhellmann/entry_point_inspector#1]
When attempt to run `ep show` commands, I get an error **'generator' object has no attribute 'next'**:

```
$ epi ep show epi.commands ep_show
'generator' object has no attribute 'next'

$ epi -v ep show epi.commands ep_show
found extension EntryPoint.parse('value = cliff.formatters.value:ValueFormatter')
found extension EntryPoint.parse('yaml = cliff.formatters.yaml_format:YAMLFormatter')
found extension EntryPoint.parse('shell = cliff.formatters.shell:ShellFormatter')
found extension EntryPoint.parse('json = cliff.formatters.json_format:JSONFormatter')
found extension EntryPoint.parse('table = cliff.formatters.table:TableFormatter')
Looking for ep_show in group epi.commands
'generator' object has no attribute 'next'
```

I am running Python version 3.3.5 (Continuum Analytics, Inc.; Anaconda).

My other dependencies are:

```
entry-point-inspector==0.1
  - cliff [installed: 1.15.0]
    - prettytable [required: <0.8, installed: 0.7.2]
    - six [required: >=1.9.0, installed: 1.10.0]
    - PyYAML [required: >=3.1.0, installed: 3.11]
    - unicodecsv [required: >=0.8.0, installed: 0.14.1]
    - argparse [installed: 1.4.0]
    - pyparsing [required: >=2.0.1, installed: 2.0.6]
    - cmd2 [required: >=0.6.7, installed: 0.6.8]
      - pyparsing [required: >=2.0.1, installed: 2.0.6]
    - pbr [required: <2.0, installed: 1.8.1]
    - stevedore [required: >=1.5.0, installed: 1.9.0]
      - argparse [installed: 1.4.0]
      - six [required: >=1.9.0, installed: 1.10.0]
      - pbr [required: >=1.6, installed: 1.8.1]
```

**Comment by dhellmann** [comment]
This is probably an issue with the version of python involved. I'll have to give a look at using six to abstract the generator syntax.

**Comment by dhellmann** [comment]
This should have been fixed by #4
